env/env.cc

```diff
@@ -618,13 +638,63 @@ Status Env::LoadEnv(const std::string& value, Env** result,
     *result = env;
   }
 #else
+  (void)config_options;
   (void)result;
   (void)guard;
   s = Status::NotSupported("Cannot load environment in LITE mode", value);
 #endif
   return s;
 }
 
+Status Env::CreateFromSystem(const ConfigOptions& config_options, Env** result,
```

Comment:
CreateFromSystem is a hard to understand. Maybe use a better name?

Reply:
I am open to a better name.  Got any suggestions?  I was going to call it CreateFromEnv but that is a little misleading as well...